--- a/ugrd/initramfs_generator.py
+++ b/ugrd/initramfs_generator.py
@@ -24,7 +24,7 @@ class InitramfsGenerator(NestConfig):
         self.build_tasks = ["build_enum", "build_pre", "build_tasks", "build_late", "build_deploy", "build_final"]

         # init_pre and init_final are run as part of generate_initramfs_main
-        self.init_types = ["init_debug", "init_main", "init_mount"]
+        self.init_types = ["init_debug", "init_unlock", "init_main", "init_mount"]

         # Passed kwargs must be imported early, so they will be processed against the base configuration
         self.config_dict.import_args(kwargs)
